rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "1.76.0"
+channel = "1.78.0"
```

Comment:
on upgrading this wasm bindgen starts failing, did you check this ?

Reply:
@Datron 
there are 5 more places where you need to change it to `1.78.0` in 4 files